packages/api/src/services/library_item.ts

```diff
@@ -1039,7 +1039,10 @@ export const updateLibraryItemReadingProgress = async (
   }
 
   const updatedItem = result[0][0]
-  await pubsub.entityUpdated<ItemEvent>(EntityType.ITEM, updatedItem, userId)
+  if (updatedItem.readingProgressBottomPercent === 100) {
```

Comment:
Here I try to reduce the number of PAGE_UPDATE event being sent so we only send the event when the item is fully read or reset to unread